org/cloudfoundry.yml

```diff
@@ -2977,10 +2981,13 @@ orgs:
           App Runtime Interfaces WG Autoscaler:
             description: Approvers for the App Runtime Interfaces Autoscaler area
             maintainers:
+            - asalan316
             - bonzofenix
             - Gerg
-            - silvestre
             - joergdw
+            - mvach
+            - salzmannsusan
+            - silvestre
```

Comment:
Do the new maintainers meets the requirements described in https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0006-approver-requirements.md?

Reply:
Thanks for bringing this up: Moved mvach and salzmannsusan to the members section until they meet the requirements.